Makefile

```diff
@@ -20,11 +20,14 @@ LD_FLAGS="-w -X $(REPO_PATH)/version.Version=$(VERSION)"
 # Dependency versions
 GOLANGCI_VERSION = 1.21.0
 
-build: bin/dex bin/grpc-client
+build: bin/dex bin/example-app bin/grpc-client
```

Comment:
I'd recommend leaving it as its own target and not making it required whenever a regular `build` target is being called.

Reply:
As you wish.
But I will then instead add a step `make bin/example-app` to `ci.yml`, to ensure that the example app builds correctly.

Reply:
That's actually a good idea. You could create an 'examples' target that builds all example binaries (including grpc-client)